app-backend/common/src/main/scala/S3.scala

```diff
@@ -217,6 +224,16 @@ final case class S3(
   def putObject(putObjectRequest: PutObjectRequest): PutObjectResult =
     client.putObject(putObjectRequest)
 
+  def putObjectMultiPart(bucket: String, key: String, file: File) = {
```

Comment:
It could be nice if you could add the explicit return type here (`Unit`), to make the declaration more clear for the future versions of us.

Reply:
Agreed. I will add this in the next commit.